tests/prov/test_prov_k8s_cortx_upgrade.py

```diff
@@ -372,3 +378,119 @@ def test_42179(self):
         assert_utils.assert_true(resp)
         self.collect_sb = False
         LOGGER.info("--------- Test Completed ---------")
+
+    @pytest.mark.lc
+    @pytest.mark.cortx_upgrade
+    @pytest.mark.tags("TEST-41951")
+    def test_41951(self):
+        """Verify suspend already suspended upgrade."""
+        pull_image_thread_list = []
+        LOGGER.info("Test Started.")
```

Comment:
Please move the get installed version and compare version to setup_method from all the test.